instructions/instructors.md

```diff
@@ -0,0 +1,310 @@
+# Supporting Learners with Carpentries Scaffolds
+
+This document is for Instructors who are looking to provide solutions for
+learners who have had trouble installing the required software for official
+Carpentries lessons. We have designed these resources (that we are calling
+Scaffolds) to minimize the effort required for you and the learner. We will
+cover the following topics here:
+
+ - [Platforms](#platforms) 
+    - [RStudio Cloud](#rstudio-cloud)
+    - [My Binder](#my-binder)
+ - [Links to lessons](#links-to-lessons)
+ - [By Language](#by-language)
+    - [Git](#git)
+    - [Python](#python)
+    - [R](#r)
+    - [Shell](#shell)
+    - [OpenRefine](#openrefine)
+
+# Platforms
+
+We recommend using one of two platforms for your learners depending on the
+lesson type: [RStudio Cloud][rs-cloud] and [My Binder][binder]. Both of these
+services are free and have default configurations that are identical or nearly
+identical to the interfaces the users will have on their own machines. We will
+outline their benefits and challenges in the next two sections. 
+
+## RStudio Cloud
+
+[RStudio Cloud][rs-cloud] is a service from [RStudio](https://rstudio.com),
+which is an online RStudio session that is currently free. This solution has
+been used in university classroom contexts for a couple of years and is well maintained.
+For more information, see this presentation: <https://rstd.io/cloud-classroom>.
+
+Advantages:
+
+ - Consistent interface
+ - Rapid setup (< 2 minutes)
+ - Secure interface
+ - Integrated terminal with git installed
+ - Rapid package installation
+ - Persistent data storage
+ - Projects are private by default
+ - Projects can be templated and shared
+
+Drawbacks:
+ 
+ - **No nano text editor**
```

Comment:
add "(use RStudio as the text editor)" ?

Reply:
Done!